apps/core/src/dashboards/dashboards.controller.ts

```diff
@@ -112,10 +114,41 @@ export class DashboardsController {
       throw either.err;
     }
 
-    if (!either.ok) {
+    if (isNothing(either.ok)) {
       throw new NotFoundException();
     }
 
     return either.ok;
   }
+
+  @HttpCode(200)
+  @Delete('bulk')
+  public async bulkDelete(
+    @Body() bulkDeleteDashboardsDto: BulkDeleteDashboardDto,
+  ): Promise<{ deletedIds: Dashboard['id'][] }> {
+    // cache is invalid after deleting a dashboard
```

Comment:
why is it invalid after deleting a dashboard?

Reply:
The cache could contain dashboards which have been deleted. We currently aren't working with the cache in a granular manner and are just nuking it on changes. The better method would be to mutate the cache to remove the deleted dashboards. We're probably gonna remove the cache though, as it won't work if we have multiple application instances, so not something for us to deal with now.